File: problem/5/perl/solve_5.pl
#!/usr/bin/env perl6
# http://projecteuler.net/problem=5

use v6;

my @numbers_to_divide_by = (11..19);

loop (my $x = 33522128640; $x > 0 ; $x -= 20) {
        my $success = True;
        for @numbers_to_divide_by -> $y {
                unless $x % $y == 0 {
                        $success = False;
                        last;
                }
        }
        if ($success) {
                say $x;
        }
}

File: problem/5/python/solve_5.py
#!/usr/bin/env python
# http://projecteuler.net/problem=5

numbers_to_divide_by = range(11,20)

x = 33522128640

while x > 0:
        success = True
        for y in numbers_to_divide_by:
                if not x % y == 0:
                        success = False
                        break
        if success:
                print x
        x -= 20
